api/ws/rpc/failureCodes.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
```

Comment:
here I would like to describe the reasons why handshake can fail - these won't be a code errors occurred during a handshake process